package.json

```diff
@@ -40,12 +40,12 @@
     "suitcss-utils-display": "^0.4.0"
   },
   "scripts": {
-    "init": "npm run module && npm run virtualenv && npm run pip && npm install && npm run config",
+    "setup": "npm run module && npm run virtualenv && npm install && npm run config",
```

Comment:
> @miketaylr already in #1314 as a bullet point, first comment.

Cool. But let's s/init/setup/ before merging. And #1176 can probably be closed at that point.

(sudo can happen later/before/doesn't matter to me)